src/pages/edge/lifecycle-for-edge-network/migration-reference.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Mobile Core Lifecycle for Edge Network migration reference
+description: A reference for understanding the differences between Lifecycle for Edge Network vs Lifecycle for Analytics.
+keywords:
+- Lifecycle
+- Lifecycle V2
+- Lifecycle for Mobile Core
+- Lifecycle for Platform
+- Lifecycle for Edge Network
+- Lifecycle Edge
+- Lifecycle iOS
+- Lifecycle Android
+- Lifecycle for Analytics
+- Product reference
+---
+
+# Lifecycle for Edge Network migration reference
+
+The Lifecycle for Edge Network Mobile SDK extension enables collecting application lifecycle data including launches, installs, upgrades, and closes in addition to application and device dimensions such as the application identifier and device name and model. When used together with the Mobile SDK Edge Network extension, Lifecycle events are sent to the Adobe Experience Platform Edge Network on each launch and close of the application.
+
+The Lifecycle for Edge Network extension is built into the existing Lifecycle extension bundled with Mobile Core and introduces new events to capture when an application is launched and closed. These events are separate and distinct from the Lifecycle events handled by the Analytics extension and are dispatched from different triggers and contain different metrics and dimensions. Both "Lifecycle for Edge Network" and "Lifecycle for Analytics" serve similar functions but behave in different ways. The changes made to the Lifecycle extension to support the Edge Network were done, in part, to support AJO and CJA features and provide greater flexibility at reporting time. The following outlines these changes to give you an understanding of the changes to your data when migrating from Lifecycle for Analytics to Lifecycle for Edge Network.
+
+## Installing Lifecycle for Edge Network
+
+The Mobile SDK Lifecycle for Edge Network extension is used together with the Mobile SDK Edge Network extension, and both must be included and registered in an application. A rule is required, however, to forward the Lifecycle events to the Edge Network extension. To configure and install Lifecycle for Edge Network, follow the instructions from the [overview page](../lifecycle-for-edge-network/index.md#configure-and-install-lifecycle-for-edge-network) to set up your XDM schema, add a rule to forward the Lifecycle Application Launch and Application Close events to the Edge Network, add the Edge Network extension to your application, and add Lifecycle and the required API calls to your application if not already done. The Lifecycle for Edge Network does not add any new APIs so application changes to migrate to the Edge Network from Analytics are minimal. To disable Lifecycle for Analytics, simply remove the Analytics extension from your application.
```

Comment:
This link can also just be `./index.md#configure-and-install-lifecycle-for-edge-network`.